### PR TITLE
Add social profiles of 10 speakers: solves #162

### PR DESCRIPTION
**Pull request by kubami9**
Solves #162
**Add social profiles of 10 speakers:**
 - Praveen Patil: github, twitter
 - William Hooi: medium
 - Mhairi O'Hara: linkedin
 - Barry Threw: linkedin
 - Damini Satya Kammakomati: twitter
 - David Effendi: github, linkedin
 - Huiren Woo: github, twitter
 - Sayanee Basu: github, twitter, linkedin
 - Satoshi Nagayasu: medium, github, twitter, linkedin
 - Sher Minn Chong: github

**Preview link: https://kubami9.github.io/2016.fossasia.org-km-contribute/**

### CODE REVIEW

**Review by akshatnitd — COMMENTED** [review]
Please provide a deployment link

**Comment by kubami9** [comment]
@akshatnitd Preview link: https://kubami9.github.io/2016.fossasia.org-km-contribute/

File: index.html [hunk 1]
```diff
@@ -748,8 +748,8 @@ <h1>Speakers</h1>
 										</div>
 										<div class="social-links vertical-align">
 											<a href="https://swarajyamag.com/author/17549"><i class="icon fa fa-external-link"></i></a>
-											<a href="#"><i class="icon fa fa-github"></i></a>
-											<a href="#"><i class="icon social_twitter"></i></a>
+											<a href="https://github.com/wavicles"><i class="icon fa fa-github"></i></a>
+											<a href="https://twitter.com/_gnovi"><i class="icon social_twitter"></i></a>
 											<a href="https://linkedin.com/in/praveen-patil-5771a631"><i class="icon social_linkedin"></i></a>
 											<a href="https://twitter.com/5Forty3"><i class="icon social_twitter"></i></a>
```
Comment (akshatnitd) [inline index.html]: Replace twitter link here

File: index.html [hunk 2]
```diff
@@ -1320,9 +1320,9 @@ <h1>Speakers</h1>
 										<div class="social-links vertical-align">
 											<a href="https://twitter.com/Daminisatya"><i class="icon fa fa-external-link"></i></a>
```
Comment (akshatnitd) [inline index.html]: Remove this duplicate twitter link

**Review by akshatnitd — COMMENTED** [review]
Please make requested changes. Also, in some places I can see there are more than 1 links of a particular website (eg : 2 twitter icons), it would be great if you could remove them :)

**Comment by kubami9** [comment]
@akshatnitd fixed and squashed 😉

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**